helpers/readPosts.js

```diff
@@ -24,7 +24,7 @@ const readPosts = (callback) => {
         const posts = JSON.parse(file);
 
         // return the object to the callback
-        callback(error, posts);
+        //callback(error, posts);
```

Comment:
Try not to leave commented code in pull requests once you have changed it. Comments are normally reserved for documentation only. Here is a good article explaining that: https://medium.freecodecamp.org/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83